README.md

```diff
@@ -68,3 +68,26 @@ you want is being brought into your application.
 * `ember build`
 
 For more information on using ember-cli, visit [http://ember-cli.com/](http://ember-cli.com/).
+
+# Linking This Addon For Local Testing
+
+## Linking
+
+1. From command line at the root of this project run the `npm link` command
+to _link_ this addon within your local node repository.
+1. From the _other_ Ember project that you wish to test the addon in, execute
+the following command:
+`npm link ember-cli-font-awesome-sass`.
+1. Now in that same _other_ Ember project, you should go into the
+`package.json` and add the ember addon with the version _*_.  It will
+look something like this: `"ember-cli-font-awesome-sass": "*"`.  Now
+when/if you execute `npm install` on this other project it will know
+that your addon is linked.
+
+## Unlinking
+
+1. Remove the addon from your local node repository:
+`npm uninstall -g ember-cli-font-awesome-sass`
+1. Remove the reference to the `ember-cli-font-awesome-sass` in your _other_
+project's `package.json`.
+1. Run an `npm prune` from the root of your _other_ project's command line.
```

Comment:
This linking section is loosely documented in the _ember-cli_ user guide...I've gone a bit further here so I can help myself remember these steps.